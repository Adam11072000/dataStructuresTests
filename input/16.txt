init 5831
resetLabelScore 723 -1 6954
addImage 3044
addImage 2943
addImage 1440
quit