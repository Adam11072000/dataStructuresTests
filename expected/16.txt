init done.
resetLabelScore: INVALID_INPUT
addImage: SUCCESS
addImage: SUCCESS
addImage: SUCCESS
quit done.
